lib/vsc/install/shared_setup.py

```diff
@@ -1565,12 +1581,15 @@ def prepare_rpm(self, target):
         if pkgs is not None:
             getattr(__builtin__, '__target')['excluded_pkgs_rpm'] = pkgs
 
+        # Locate the readme and store its path
+        self.readme = self.locate_readme()
+
         # Add (default) and excluded_pkgs_rpm packages to SHARED_TARGET
         # the default ones are only the ones with a __init__.py file
         # therefor we regenerate self.package files with the excluded pkgs as extra param
         self.package_files = self.files_in_packages(excluded_pkgs=pkgs)
         _fvs('prepare_rpm').SHARED_TARGET['packages'] = self.generate_packages()
-        self.build_setup_cfg_for_bdist_rpm(target)
+        self.build_setup_cfg_for_bdist_rpm(target, self.readme)
```

Comment:
why do you pass a an attribute to a method?

Reply:
The problem is that `build_setup_cfg_for_bdist_rpm` is a static method, so it does not have access to object attributes. I did not want to change the scope of the method in case something breaks somewhere else, so I just passed the attribute to the function. I agree that it's not pretty though.
https://github.com/hpcugent/vsc-install/pull/176/commits/d5629a46df9c3e812547e56c5dd649bf3a8e1c8e changes the approach and uses the `long_description` in target to store the readme file path.